.github/workflows/windows-msys2.yml

```diff
@@ -75,41 +75,39 @@ jobs:
       - name: Build GnuCOBOL
         shell: msys2 {0}
         run: |
-          cd _build
-          make --jobs=$(($(nproc)+1))
+          make -C _build --jobs=$(($(nproc)+1))
 
-      - name: Install GnuCOBOL
+      - name: Run testuite
         shell: msys2 {0}
         run: |
-          cd _build
-          make install
-          find /opt/cobol > install.log
+          sed '/AT_SETUP(\[temporary path invalid\])/a \
+                AT_SKIP_IF(\[true\])' \
+              -i tests/testsuite.src/used_binaries.at
+          make -C _build/tests check TESTSUITEFLAGS="--jobs=$(($(nproc)+1))"
+          make -C _build/tests test
 
-      - name: Upload install-${{ matrix.target }}.log
+      - name: Upload testsuite-${{ matrix.target }}.log
         uses: actions/upload-artifact@v4
         with:
-          name: install-${{ matrix.target }}.log
-          path: ${{ env.GITHUB_WORKSPACE }}/_build/install.log
+          name: testsuite-${{ matrix.target }}.log
+          path: ${{ env.GITHUB_WORKSPACE }}/_build/tests/testsuite.log
 
-      - name: Run testuite
+      - name: Install GnuCOBOL
         shell: msys2 {0}
         run: |
-          sed -i '/AT_SETUP(\[temporary path invalid\])/a AT_SKIP_IF(\[true\])' tests/testsuite.src/used_binaries.at
-          cd _build/tests
-          make check TESTSUITEFLAGS="--jobs=$(($(nproc)+1))"
-          make test
+          make -C _build install
+          find /opt/cobol > _build/install.log
 
-      - name: Upload testsuite-${{ matrix.target }}.log
+      - name: Upload install-${{ matrix.target }}.log
         uses: actions/upload-artifact@v4
         with:
-          name: testsuite-${{ matrix.target }}.log
-          path: ${{ env.GITHUB_WORKSPACE }}/_build/tests/testsuite.log
+          name: install-${{ matrix.target }}.log
+          path: ${{ env.GITHUB_WORKSPACE }}/_build/install.log
```

Comment:
Is there any reason to do the install and provide the install log here? There is no artifact using that and we do have the distmgw for a "ready-to-use" package.

Reply:
That's something Fabrice did a long time ago, we just kept that until now...

Reply:
as the CI is now stable "in general" it seems like a good time to adjust then :-)